sdk/cosmos/azure-spring-data-cosmos/README.md

```diff
@@ -3,8 +3,18 @@
 **Azure Spring Data Cosmos** provides Spring Data support for Azure Cosmos DB using the [SQL API][sql_api_query], based on Spring Data framework.
 [Azure Cosmos DB][cosmos_introduction] is a globally-distributed database service which allows developers to work with data using a variety of standard APIs, such as SQL, MongoDB, Cassandra, Graph, and Table.
 
-## Spring data version support
-This project supports both `spring-data-commons 2.2.x` and `spring-data-commons 2.3.x` versions. Maven users can inherit from the `spring-boot-starter-parent` project to obtain a dependency management section to let Spring manage the versions for dependencies.
+## Spring Boot Support Policy
+This project supports multiple [Spring Boot Versions][spring_version_mapping]. For complete list of currently supported versions, please visit our [Spring Version Mapping][spring_version_mapping].
+
+Spring Boot releases are marked as "End of Life" when they are no longer supported or released in any form. If you are running an EOL version, you should upgrade as soon as possible. 
+
+Please note that a version can be out of support before it is marked as "End of Life". During this time you should only expect releases for critical bugs or security issues.
+
+For more information on Spring Boot supported versions, please visit [Spring Boot Supported Versions][spring_boot_supported_versions].
+
+## Spring Boot Version Support
+Maven users can inherit from the `spring-boot-starter-parent` project to obtain a dependency management section to let Spring manage the versions for dependencies.
+
```

Comment:
Should we still mention or stick to the Spring Data versions here, because the Spring Data Cosmos is in the Spring Data project https://github.com/spring-projects/spring-data-commons/wiki. But I understand that most users will use Spring Boot, we can add another section for Spring Boot version support